### PR TITLE
build errors fixes

### PR DESCRIPTION
**Pull request by mobilunity-user**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -45,10 +45,10 @@
     "gulp-sort": "^2.0.0",
     "gulp-transform": "^1.1.0",
     "gulp-typescript": "^3.1.5",
-    "gulp-uglify-harmony": "https://github.com/jiborobot/gulp-uglify-harmony.git#master",
+    "gulp-uglify-harmony": "git+https://github.com/jiborobot/gulp-uglify-harmony.git#master",
     "mocha": "^3.2.0",
     "rimraf": "^2.6.1",
-    "typescript": "^2.2.1",
+    "typescript": "^2.3.4",
```
Comment (gregolsky) [inline package.json]: Just `2.3.4` - without the dash `^`.

Comment (gregolsky) [inline package.json]: @a-serdukov-mobilunity We have to set "typescript": "2.3.x". Otherwise it's still going to install 2.4 for node 8.

**Review by gregolsky — CHANGES_REQUESTED** [review]

File: package.json (hunk 1 above)
Comment (mobilunity-user) [reply package.json]: done

File: package.json [hunk 2]
```diff
@@ -45,10 +45,10 @@
     "gulp-sort": "^2.0.0",
     "gulp-transform": "^1.1.0",
     "gulp-typescript": "^3.1.5",
-    "gulp-uglify-harmony": "https://github.com/jiborobot/gulp-uglify-harmony.git#master",
+    "gulp-uglify-harmony": "git+https://github.com/jiborobot/gulp-uglify-harmony.git#master",
     "mocha": "^3.2.0",
     "rimraf": "^2.6.1",
-    "typescript": "^2.2.1",
+    "typescript": "2.3.*",
```
Comment (gregolsky) [inline package.json]: @a-serdukov-mobilunity  Almost :) `*` means *any* version. Make it `~2.3.4`, which means only *patch* number (last one) can increment.

**Review by gregolsky — CHANGES_REQUESTED** [review]

File: package.json (hunk 2 above)
Comment (mobilunity-user) [reply package.json]: ok

**Pull request merged**